### PR TITLE
add style

### PR DESCRIPTION
**Pull request by JasemDuncan**
In this milestone, we add a style sheet for the awesome books application.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+}
+
+.full-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: aliceblue;
+}
```
Comment (EricMbouwe) [inline style.css]: - [ ] Your application should match this [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png)
Screenshot from 2021-10-21 17-40-31


Here is what your app look like
Screenshot from 2021-10-21 17-42-34
- No background on each book item
- No divider between the book list and the new book form

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @JasemDuncan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (rloterh) [reply style.css]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Jasem & Bunbee, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**